app/views/application/api.html.haml

```diff
@@ -18,7 +18,7 @@
       %p
         BIP provides HTTP REST-style API which uses JSON to represent returned
         resources. Full access to the API is allowed for all registered users.
-        In order to authenticate users pass <code>api_key</code> parameter in
+        In order to authenticate users pass <code>X-BIP-Api-Key</code> header in
```

Comment:
I think the controller part does still not exclude passing api key as a param. But I understand we want to encourage passing it in a header?